app/assets/stylesheets/styles/post-edit.scss

```diff
@@ -10,12 +10,12 @@ label {
   margin: 0;
 }
 
-#container {
+#edit-container {
   display: grid;
   grid-template-columns: 12.5% 75% 12.5%;
 }
 
-#header-container {
+#edit-header-container {
```

Comment:
Use a class

Reply:
This element was given the class editHeaderContainer and the id was removed